home/templates/home/questions/new_question.html

```diff
@@ -0,0 +1,16 @@
+{% extends "home/base.html" %}
+{% load static %}
+{% block content %}
+
+<body class="landing-bg">
```

Comment:
About the class landing-bg. it's a very light color used for the landing page.
I think you should change it to the 'main-bg' class so it will match the design in the Wireframe, and have
identical bg color to the display question page and the tags page.
Also, I think giving the body tag this class should be sufficient and you don't need to write it also in the div below

Reply:
Changed